# face-recognisation
it only recognizes the face but not identifies it

you will need to change the location of frontal_face_default according to the location in ur desktop. Cnage it to the location where it is saved in ur Python37-32\Lib\site-packages\cv2\data\haarcascade_frontalface_default and also the image u want to  test. 
